config/routes.rb

```diff
@@ -6,6 +6,7 @@
 
   resources :episodes do
     resources :announcements
+    get '/download', to: 'episodes#download', on: :member
```

Comment:
It's already `on: :member`
and it's enough just 
```ruby
get 'download'
```
to apply all you write here

Reply:
Ok, I'll redo it now